Fhi.HelseId.Web/HelseIdWebKonfigurasjon.cs

```diff
@@ -47,9 +47,12 @@ protected override IEnumerable<string> FixedScopes
                 "profile",
                 "helseid://scopes/identity/pid",
                 "helseid://scopes/identity/pid_pseudonym",
-                "helseid://scopes/identity/security_level",
-                "helseid://scopes/hpr/hpr_number"
+                "helseid://scopes/identity/security_level"
             };
+            if (UseHprNumber)
```

Comment:
Do we know if we have any consumers who is using HPR number and has this disabled? In those cases we can argue that is a bug, but I want to nevertheless bring this up to highlight it.

Reply:
Yes, we do.  The complaint came from a dev.